_content/av/geometry-study_kawae-yuki.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Geometry Study: Pattern Explorations"
+authors:
+  - "Yuki Kawae"
+sexternal_url: "https://www.youtube.com/watch?v=yj2L07KwL-c"
+course: japanese
+tags:
+  - art
+  - bart
+year: 2022
+month: feb
+minutes: 7
+---
+
+In this video, artist and designer Yuki Kawae showcases a handful of Zen garden patterns in sand.
```

Comment:
Go ahead and add that interview as a link in the description.  Do it in a second paragraph under this one.